inst/scripts/query-pubmed-grants.R

```diff
@@ -64,8 +65,9 @@ source(glue::glue("{base_dir}/R/global-hard-coded-variables.R"))
 
 # Login to synapse
 ## Synapse client and logging in
+reticulate::use_condaenv("r-reticulate", required=TRUE)
 synapseclient <- reticulate::import("synapseclient")
-syntab <- reticulate::import("synapseclient.table")
+# syntab <- reticulate::import("synapseclient.table")
 syn <- synapseclient$Synapse()
 if (!is.na(opts$auth_token)) {
   syn$login(authToken = opts$auth_token)
```

Comment:
Future improvement: this isn't necessary since the R client will automatically read in `SYNAPSE_AUTH_TOKEN` env variable.